harness/determined/cli/trial.py

```diff
@@ -138,33 +138,43 @@ def output_logs(args: Namespace):
     bundle = None
     if args.output_dir is None:
         output_dir = os.getcwd()
-        formatted_dt = datetime.now().strftime('%Y-%m-%d_%H:%M:%S')
-        tar = f"det-bundle-trail_{args.trial_id}-{formatted_dt}.tar.gz"
-        fullpath = os.path.join(output_dir, tar)
-        bundle = tarfile.open(fullpath, "w:gz")
 
     else:
         bundle = tarfile.open(f"{args.output_dir}.tar.gz", "w:gz")
```

Comment:
This should be:

```
output_dir = args.output_dir
if output_dir is None:
    output_dir = os.getcwd()
...
```

Reply:
yes, sorry I didn't commit that change correctly.